src/components/button/Button.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from "prop-types";
+import tw from "twin.macro";
+import theme from "@/themes/theme";
+import styled from "styled-components";
+
+const StyledButton = styled.button(
+  ({ className, styles: { block, color, border, shape, size } = {} }) => [
+    theme.button.default.base,
+    theme.button.default.color,
+    theme.button.default.hover,
+    theme.button.default.focus,
+    block && tw`w-full h-full`,
+    color && theme.button.color[color],
+    border === "gray" && theme.button.border[border],
```

Comment:
Border has more options then just `gray`.  There is the following:

* gray
* grey
* transparent
* primary
* success
* danger

And whatever option is chosen we need to add in `base` (see my comments for theme.js)

Reply:
fixed